package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "telemetry-events",
-  "version": "1.0.1",
+  "version": "1.1.0",
```

Comment:
In order to publish the updated module, we need to change the version. This is not a major version because we're not breaking anything.

That leaves us with "new feature" or "bug fix". We both, said that it _could_ be looked at as a new feature. If we agree that it could be interpreted as a new feature (and it could be interpreted as a bug fix), then to me, a minor version bump is appropriate because it subsumes patch version bump.